docker-compose/dse-6.8/docker-compose.yml

```diff
@@ -49,12 +49,11 @@ services:
     networks:
       - stargate
     ports:
-      - 9042:9042
-      - 8080:8080
-      - 8081:8081
-      - 8083:8083
```

Comment:
Is it intentional that 8083 is removed here?

Reply:
Yes, it was originally thought that the docs API port would be `8083` but is now apparently `8180`. I will add that to the PR description.